### ISSUE TITLE
Font Awesome JS not loading correctly

### ISSUE THREAD

**Issue by alexanderluiscampino** [FortAwesome/Font-Awesome#17622]
**Describe the bug**
The following link is not loading correctly `https://kit.fontawesome.com/a076d05399.js`

**To Reproduce**
Just try to load  `https://kit.fontawesome.com/a076d05399.js` as a webpage and a weird error message will appear. Sometimes it works, most of the times it does not. 

**Expected behavior**
The JS for the fonts should just load

**Screenshots**
![image](https://user-images.githubusercontent.com/32500355/107566089-44995580-6b99-11eb-9ff9-596fbb09db09.png)
**Version and implementation**
Version: Not sure
Browser and version: Latest Chrome Browser

- [ ] SVG with JS
- [x] Web Fonts with CSS
- [ ] SVG Sprites
- [ ] On the Desktop

**Bug report checklist**
- [x] I have filled out as much of the above information as I can
- [x] I have included a test case because my odds go _way_ up that the team can fix this when I do
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Is this a Font Awesome Pro kit?

**Comment by alexanderluiscampino** [comment]
Hi,

This is not part of the Pro Kit. At least, hasn't been for the last 1.5 years. Has something changed?

**Comment by tagliala** [comment]
Under kit settings at https://fontawesome.com/kits/a076d05399/settings

could you please confirm that:
- this is a free kit
- your domain is allowed

**Comment by tagliala** [comment]
Oh, got it.

Did you get the code from https://www.w3schools.com/icons/fontawesome5_intro.asp ?

Developers are supposed to create your own kit, and use their own code. I will try to contact them, there is a lot of website using the same kit code

**Comment by tagliala** [comment]
Official answer from the team:

> It looks like you were using a kit code from the w3schools.com web site. That kit code is intended to be used on w3schools.com and they a referrer check now on their kit. You can get your own free kit code here https://fontawesome.com/start. After getting our free kit code you can just replace the one from w3schools.com and everything should load just fine.

**Comment by mbuotim** [comment]
hi thanks for this insight, i did everything correctly but mine is still not working

**Comment by abtheprogrammer** [comment]
I read #17622 comment and solve my error in 2 mins
Tysm!

**Comment by noshor** [comment]
> 

Thank you @tagliala, Unfortunately, we are having the same issue with our own kit, the kit is limited to our own domain using noshor.com and *.noshor.com but it showing the same CORS error, could you please advise? thank you so much!

**Comment by tagliala** [comment]
Could you please provide a reproducible test case?

**Comment by noshor** [comment]
> 

Thank you for the prompt reply @tagliala, FYI, we figured out the issue is related to the JS-free embed method, while embedding the kit using the JS embed code worked, we were initially trying to embed the kit using the CSS file as opposed to the JS script. To reproduce the issue, all you need to do is to embed using the CSS embed code instead of the JS embed code and you will be able to see the CORS error, please let me know if you need more information, happy to share a link to where that exists if that's helpful, thank you.

**Comment by tagliala** [comment]
> To reproduce the issue, all you need to do is to embed using the CSS embed code instead of the JS embed code and you will be able to see the CORS error, please let me know if you need more information, happy to share a link to where that exists if that's helpful, thank you.

Sorry, I cannot confirm: https://jsfiddle.net/tagliala/ryuf9kdb/2/

Please fill out a [bug report template for web applications](https://github.com/FortAwesome/Font-Awesome/issues/new?assignees=&labels=bug%2Cneeds-triage&template=100_web_bug_report.yml&title=Bug%3A+)

**Comment by DekkerEdwin** [comment]
On Safari I had to add www.mydomainname.com in the allowed domains. Having mydomainname.com in the allowed domains was not sufficient.

**Comment by bodoszidi** [comment]
Hi
Im having a problem with my kit js file not loading on localhost. Giving an 403 error.
I attempted to add localhost to domains but says we dont need to because its automatically added for testing purposes.
What else can I do to get it working locally?

**Comment by robmadole** [comment]
@bodoszidi if you are a paid customer you can email support hello@fontawesome.com. Otherwise, the most common reason is a missing `Origin` header due to it being blocked by [Referrer Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Referrer-Policy) or [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Guides/CSP)
